### PR TITLE
fix: menu button interactions state

### PR DESCRIPTION
**Pull request by a-m-dev**
basically fixes this issue:

https://github.com/qlik-oss/nebula-table-utils/assets/29652890/18f17498-6074-474b-ac83-347e98cc39ec

### CODE REVIEW

File: src/components/HeadCellMenu/HeadCellMenu.tsx [hunk 1]
```diff
@@ -76,6 +76,7 @@ const HeadCellMenu = ({
         aria-expanded={open ? 'true' : undefined}
         aria-haspopup="true"
         data-testid="nebula-table-utils-head-menu-button"
+        isInteractionsActive={searchRelatedArgs?.interactions.active ?? false}
```
Comment (cbt1) [inline src/components/HeadCellMenu/HeadCellMenu.tsx:79]: Maybe it time to move `interactions` out from `searchRelatedArgs`? As it defines if the menu can be opened or not

**Review by cbt1 — APPROVED** [review]
LGTM, just a minor improvement comment

File: src/components/HeadCellMenu/HeadCellMenu.tsx (hunk 1 above)
Comment (a-m-dev) [reply src/components/HeadCellMenu/HeadCellMenu.tsx:79]: Sure, sounds legit, I'll create a PR for that in future

**Pull request merged**